### DISCUSSION TITLE
Error Orange de repente

### DISCUSSION THREAD

**Discussion by Cholo68** [Paco8/plugin.video.orange.spain#23] [General]
Ayer lo que me ocurrio es que fui a abrir Orange y directamente me salia error de registro. Probe a instalar y desinstalar el addon y meter la carpeta addon data que tengo guardada con mis credenciales y directamente al entrar de nuevo error de registro, asi prrobe 3 veces a desinstalar addon y nada. Al final tuve que instalar de nuevo y antes de copiar nada entré en cuentas y tuve que meter d enuevo los credenciales. Alguna vez que me pasó algo parecido con copiar la carpeta addon data fue suficiente pero esta vez no habia manera.
Sé que me dijiste en otra ocasion de enviar log, te refieres al log de kodi que se genera en la raiz no?.
Gracias

**Comment by Paco8** [comment]
Sí, el fichero kodi.log. Tienes que activar antes la opción "activar el registro de depuración" en Sistema, Registro. Luego entrar en el addon de Orange para que dé el error, y luego mandar el kodi.log.

**Comment by Cholo68** [comment]
Perfecto, gracias
